src/nvim/shada.c

```diff
@@ -10,9 +5,6 @@
 #include <stdint.h>
 #include <inttypes.h>
 #include <errno.h>
-#ifdef HAVE_UNISTD_H
-# include <unistd.h>
-#endif
```

Comment:
(And you did not mention this change in the commit message: it talks only about HAVE_BE64TOH.)